### PR TITLE
Client propagation fat updates

### PR DESCRIPTION
**Pull request by c00crane**
Some updates to the claim propagation tests including adding testing with a social (oidcLogin) client in addition to the normal RP client.

The social tests have run into some issues that will either result in product code changes or alterations of the tests.
More investigation needs to be done, but this work is on hold for the time being.

### CODE REVIEW

**Comment by c00crane** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.oidc.social_fat.claimPropagation

**Comment by c00crane** [comment]
#run-libby-bot

File: dev/com.ibm.ws.security.oidc.social_fat.claimPropagation/fat/src/com/ibm/ws/security/oidc_social/claimPropagation/fat/FATSuite.java [hunk 1]
```diff
@@ -32,6 +33,9 @@
         // specify OIDC tests
         OIDCBasicIdTokenClaimPropagationTestss.class,
 
+        // specify Social client tests
+        SocialBasicIdTokenClaimPropagationTestss.class,
```
Comment (ayoho) [inline dev/com.ibm.ws.security.oidc.social_fat.claimPropagation/fat/src/com/ibm/ws/security/oidc_social/claimPropagation/fat/FATSuite.java:37]: Nit: Both of these test classes end in double 's' (Testss)

**Review by ayoho — APPROVED** [review]

File: dev/com.ibm.ws.security.oidc.social_fat.claimPropagation/fat/src/com/ibm/ws/security/oidc_social/claimPropagation/fat/FATSuite.java (hunk 1 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.oidc.social_fat.claimPropagation/fat/src/com/ibm/ws/security/oidc_social/claimPropagation/fat/FATSuite.java:37]: good gravy - knew I couldn't type...
I'll fix that with another pull - want to get this delivered.

**Pull request merged**